src/storage/txn/actions/flashback_to_version.rs

```diff
@@ -96,7 +96,12 @@ pub fn flashback_to_version<S: Snapshot>(
             *next_lock_key = Some(key);
             break;
         }
+        // We need to rollback with lock.ts rather than using version timestamp by
+        // invoking rollback funtion
+        let write = Write::new_rollback(lock.ts, false);
```

Comment:
I think we should use `rollback_lock` function in the cleanup action as much as possible.

Reply:
Okay, also need to change the initialization of the reader to start_ts